src/enqueue/class-localization-constants.php

```diff
@@ -101,6 +118,8 @@ public function __construct(
     $this->send_email_route            = $send_email_route;
     $this->mailchimp_route             = $mailchimp_route;
     $this->mailchimp                   = $mailchimp;
+    $this->mailerlite_route             = $mailerlite_route;
+    $this->mailerlite                   = $mailerlite;
```

Comment:
```suggestion
    $this->mailerlite_route            = $mailerlite_route;
    $this->mailerlite                  = $mailerlite;
```

I think forms are using older CS where we used to align all the equal signs (like WP does). These seem to be offset by one space to the right.